angr/sim_type.py

```diff
@@ -614,7 +614,7 @@ class SimTypeFunction(SimType):
     _fields = ('args', 'returnty')
     base = False
 
-    def __init__(self, args, returnty, label=None):
+    def __init__(self, args, returnty, label=None, arg_names=[]):
```

Comment:
Please do not use a list as the default value of a function parameter since it can be modified (ref: [Common Gotchas](https://docs.python-guide.org/writing/gotchas/)).